### PR TITLE
Use Binder

### PR DESCRIPTION
**Pull request by mgrover1**
Switch back to using binder for execution now that secrets are working

### CODE REVIEW

**Comment by mgrover1** [comment]
@brian-rose @r-ford - I am testing to see if the revised secrets workflow works with this book as well

**Comment by brian-rose** [comment]
We'll need to make changes in the workflow files mirroring what's in the radar cookbook before the PR building will work. 

I'm still thinking through some of the implementation details though.

**Comment by mgrover1** [comment]
@brian-rose - gotcha! Should we hold off then? Otherwise I can help mirror the changes?

**Comment by brian-rose** [comment]
No please go ahead and experiment! If we restrict the changes to just two cookbook repos then it shouldn't be too hard to clean up any messes in the future :-)

**Comment by mgrover1** [comment]
@brian-rose - it looks like it is building correctly now. I think we can merge this + add back in the other notebooks

File: notebooks/example-workflows/esgf2-arm-comparison.ipynb [hunk 1]
```diff
@@ -215,14 +215,14 @@
    },
    "outputs": [],
    "source": [
-    "arm_username = os.getenv(\"ARM_USERNAME\")\n",
-    "arm_password = os.getenv(\"ARM_PASSWORD\")\n",
+    "arm_username = \"mgrover4\"\n",
```
Comment (brian-rose) [inline notebooks/example-workflows/esgf2-arm-comparison.ipynb:218]: It looks like you are hard-coding your credentials in the notebook here?

Comment (mgrover1) [reply notebooks/example-workflows/esgf2-arm-comparison.ipynb:218]: Oh no... I will fix that.

Comment (mgrover1) [reply notebooks/example-workflows/esgf2-arm-comparison.ipynb:218]: Thanks for pointing that out!

**Comment by mgrover1** [comment]
@brian-rose - it seems to be running as expected! The preview builds properly with the secrets this time.

**Pull request merged**